Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -6,34 +6,39 @@ public enum ActorStates
 {
 	Grounded = 0,
 	Jumping,
+	Falling,
 	Rolling,
 	Climbing
 }
 
 public class ActorPhysics : ActorComponent
 {
+	private new Transform transform;
+	private new Rigidbody rigidbody;
+
 	// States
+	#region States
 	public delegate void ActorStateMethod();
 
+	[ReadOnlyAttribute, SerializeField]
 	ActorStates currentState = ActorStates.Jumping;
 	protected ActorStateMethod CurrentStateMethod;
 	public Dictionary<ActorStates, ActorStateMethod> stateMethodMap = new Dictionary<ActorStates, ActorStateMethod>();
-	
-	// Movement
-	[Header("Movement")]
+	#endregion
+
+	#region Movement
+	[Header( "Movement" )]
 	public Vector3 inputVec = Vector3.zero;
 	public float moveSpeedMod = 1f;
 
-	protected Vector3 lastVelocity = Vector3.zero;
 	protected Vector3 moveVec = Vector3.zero;
 
-	protected float stoppingSpeed = 0.001f;
+	public float stoppingSpeed = 0.001f;
```

Comment:
This shouldn't be public. It's referenced in ActorPlayerPhysics but nowhere else.